prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -4808,19 +4808,16 @@ end subroutine readCustomisedHubbards
 
 
   !> Reads the electron dynamics block
-  subroutine readElecDynamics(node, input, geo, masses)
+  subroutine readElecDynamics(node, ctrl, geom)
```

Comment:
Passing `TElecDynamicsInp` should be preferred over the complete `TControl` as it makes the parser more modular.

Reply:
OK, will revert (requires two parts of the structure to be passed  separately, as mass is in there as well).